cms/envs/aws.py

```diff
@@ -81,6 +81,9 @@
 CELERY_DEFAULT_QUEUE = DEFAULT_PRIORITY_QUEUE
 CELERY_DEFAULT_ROUTING_KEY = DEFAULT_PRIORITY_QUEUE
 
+# Queue to use for updating grades due to grading policy change
+POLICY_CHANGE_GRADES_ROUTING_KEY = LOW_PRIORITY_QUEUE
```

Comment:
Talked with Eric, turns out that the task we want to route to this new queue is already included in `cms/celery.py`, so we are theoretically all set on this front.